package.json

```diff
@@ -4,27 +4,32 @@
   "private": true,
   "proxy": "http://localhost:8080",
   "dependencies": {
-    "date-fns": "latest",
     "@date-io/date-fns": "v1",
-    "@material-ui/core": "^4.9.10",
+    "@material-ui/core": "^4.9.11",
```

Comment:
Do you have to upgrade Material-UI?

Reply:
not really but its just a bug version update so it should be fine